content/user/payment/balance_owed.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Balance Owed Payments 
+description: Taking payment for balances owed 
+category: payment
+weight: 10 
+---
+
+Sometimes the payment you received for an order is just not enough 
+
+- the customer wanted to add items or options to the order
+- shipping was more expensive than anticipated
+- the customer requested additional services such as gift wrapping or insurance after completing their order
+
+In cases like this, you must take another action to collect payment
+
+### Option 1: Invoice the Customer 
+
+Most credit card gateways and PayPal will allow you to send the customer a payable invoice.  Keep the order in Pending status until the invoice has been paid.
+
+This image shows the [Square WebPay](/user/payment/square/) invoicing page:
```

Comment:
While it's probably fine to list this as `Square WebPay`, it's actually just `Square`, independent of which other features one is using with Square.